egs/ami/s5b/local/chain/tuning/run_tdnn_1f.sh

```diff
@@ -0,0 +1,266 @@
+#!/bin/bash
+
+# same as 1e but uses batchnorm components instead of renorm also adding
+# proportional-shrink 10, trained with 4 epochs
+
+# System               tdnn1e_sp_bi_ihmali tdnn1f_sp_bi_ihmali
```

Comment:
@danpovey Oh..... after the SWBD scripts are OK, I'll adding the command lines in these scripts together with the SWBD scripts